### DISCUSSION TITLE
Show selected tools when editor side menus are minimised.

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#23664] [Ideas]
(this is on master)
The new sound bank controls are not visible at normal scaling when scrolled to the top (you have to scroll to see everything)
![image](https://github.com/ppy/osu/assets/53081750/4493c2c2-5d45-40dc-9c06-692d302fd23a)
When compressed, there are 3 barely visible boxes instead, this does not look good.
![image](https://github.com/ppy/osu/assets/53081750/478db615-0c98-4d3a-bb74-379b67f25f94).
I think it would be nice to have just a single square that reflects the currently selected tool per section (paint edit moment). What do you think?
![image](https://github.com/ppy/osu/assets/53081750/8c2c617c-04e7-405d-a803-44d19c1fedb4)

**Comment by peppy** [comment]
Maybe we can still show the other options in some fashion (potentially just an empty very-short-box) to give context as to what would appear when you hover the control area.
